travis-ymls/ginko.travis.yml

```diff
@@ -0,0 +1,43 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : ginkgo
+# Source Repo         : https://github.com/onsi/ginkgo
+# Travis Job Link     : https://travis-ci.com/github/santosh653/ginkgo
+# Created travis.yml  : No
+# Maintainer          : Santosh Kulkarni <santoshkulkarni70@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+
+arch:
+  - amd64
+  - ppc64le
+language: go
+go:
+  - 1.14.x
+  - 1.15.x
+  - tip
+jobs:
+ exclude:
+  - go: tip
```

Comment:
Excluding older versions is fine but I don't think we should be excluding "tip"

Reply:
@seth-priya Now the code is running as suggested by you., Can you please review?

Reply:
as far as I can see we are still excluding "tip" for ppc64le, am I missing something?